package.json

```diff
@@ -44,6 +44,7 @@
     "@mui/lab": "6.0.0-beta.13",
     "@mui/material": "6.1.5",
     "@sentry/nextjs": "8.35.0",
+    "@tanstack/react-query": "5.65.1",
```

Comment:
Do you also want to add the devtools ?

Reply:
Not particularly, but it can be added later if necessary.